utils/vlog/tests/fig28-lutff-pair/golden.pb_type.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version='1.0' encoding='utf-8'?>
+<pb_type xmlns:xi="http://www.w3.org/2001/XInclude" name="PAIR" num_pb="1">
+  <clock name="CLK" num_pins="1"/>
+  <input name="I" num_pins="4"/>
+  <output name="O" num_pins="1"/>
+  <pb_type blif_model=".subckt dff" name="ff" num_pb="1">
+    <xi:include href="ff/ff.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
+  </pb_type>
+  <pb_type blif_model=".subckt lut4" name="lut" num_pb="1">
```

Comment:
I believe we should only use this include style if num_pb needs to be modified.  If the attribute list can be used "as-is", just use the straightforward xi:include.  When I see the explicit attribute form I'm asking myself ``what attribute was changed``.  Might be worth adding a comment like:
``<-- num_pb <= "2" -->``

Reply:
@kgugala

Reply:
done. Now if the script does not change anything the whole original xml is included. If there are changes, a comment is emitted e.g:
```
  <pb_type num_pb="32" name="q_a_ff">
    <!--old_name DFF old_num_pb 1-->
    <xi:include href="../fig42-dff/dff.pb_type.xml" xpointer="xpointer(pb_type/child::node())"/>
  </pb_type>
```